packages/open-api-page/server.js

```diff
@@ -35,26 +35,25 @@ function getPropsForPage(schema, params) {
   // rather than a specific operation-slug-based path as it would otherwise
   if (isSingleService) navData[0].path = ''
   // If there's no "page" param, then this is the landing page
-  const isLanding = !params || !params.page || params.page.length == 0
   const currentPath = params && params.page ? params.page.join('/') : ''
-
+  const isLanding = !params || !params.page || currentPath == ''
```

Comment:
Refactored the logic with an early return for landing pages to make things more clear.